src/main/scala/nl/knaw/dans/easy/multiDeposit/CommandLineOptions.scala

```diff
@@ -35,38 +36,42 @@ object CommandLineOptions {
     val opts = new ScallopCommandLine(conf, args)
 
     val settings = Settings(
-      appHomeDir = new File(Option(System.getProperty("app.home"))
-        .getOrElse(Properties.propOrNull("process.sip.home"))),
+      // TODO appHomeDir does a get on Option. Is there a default value for this or is failing hard
```

Comment:
The Java system property `app.home` must always be defined. If it is not, the application should simply fail. `process.sip.home` is obsolete. So, wrapping the `getProperty` call in `Option` is not necessary.